text/0075-span-thread-id.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2023-02-15
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/75
+- RFC Status: draft
+
+# Summary
+
+Enrich spans with thread id information.
+
+# Motivation
+
+As we have started to enrich profiling and performance UIs with data from the two products, we realized that we cannot precisely show what spans were executed on each thread (ex in profiling UI) or show what function frames were executed during a span (ex in performance UI). Having span thread id data would allow us to improve the user experience between the two products.
+
+# Supporting Data
+
+In profiling, we currently show the entire span tree regardless of what thread the user is viewing and in performance we default to showing the main thread. This is not very accurate and can lead to misleading our users.
+
+Metrics:
+No usage metrics, but this was requested by customers.
+
+# Options Considered
```

Comment:
Yes, we would need to make this thread safe. I'm not familiar with the SDK's enough to be able to tell how much of an investment or effort this would take